modules/installation-configuration-parameters.adoc

```diff
@@ -174,3 +192,41 @@ For more information about AWS tags,
 see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources]
 in the AWS documentation.
 |====
+endif::aws[]
+
+ifdef::azure[]
+.Additional Azure parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`machines.platform.azure.type`
+|The Azure VM instance type.
+|VMs that use Windows or Linux as the operating system. See the
+link:https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-supported-os?view=azs-1908[Guest operating systems supported on Azure Stack]
+in the Azure documentation.
+
+`machines.platform.azure.osDisk.diskSizeGB`
+|The Azure disk size for the VM.
+|Integer that represents the size of the disk in GB, for example `512`. The
```

Comment:
s/GB, for example `512`./GB. For example, `512`.

Reply:
@bobfuru, what's your source? The only guidance I can find in the IBM Style guide is that you need to use "for example." It does not provide a suggested punctuation and generally avoids creating a fragment.

Reply:
"For example" should be followed by a comma. As written, it's also a bit of a run-on. Would also be okay to write "...in GB (for example, `512`)." Although I try to avoid parentheses that aren't part of code.
I can't find a stated guideline in IBM or other guides, though most of thehir own refs use it at the start of a sentence.
But as written, it is short, to the point, and clear, so if that's a convention we use, then ignore my ^^ comments altogether. :)